apps/prairielearn/src/lib/db-types.ts

```diff
@@ -803,9 +803,7 @@ export type GroupUser = z.infer<typeof GroupUserSchema>;
 
 export const GroupRoleSchema = z.object({
   assessment_id: IdSchema.nullable(),
-  can_assign_roles_at_start: z.boolean().nullable(),
-  can_assign_roles_during_assessment: z.boolean().nullable(),
-  can_submit_assessment: z.boolean().nullable(),
```

Comment:
This column was never used anywhere (not even in sync), so I took the opportunity to drop it as well.